pkg/controllers/user/nodesyncer/nodessyncer.go

```diff
@@ -587,9 +580,9 @@ func aggregateRequestAndLimitsForNode(pods []*corev1.Pod) (map[corev1.ResourceNa
 		//podName -> req/limit -> data
 		for _, pod := range pods {
 			podsData[pod.Name] = make(map[string]map[corev1.ResourceName]resource.Quantity)
-			requests, limits := getPodData(pod)
-			podsData[pod.Name]["requests"] = requests
-			podsData[pod.Name]["limits"] = limits
+			request, limit := getPodData(pod)
```

Comment:
I don't agree with this change

Reply:
Fixed shadowing but it does make the code less clear than before.